lib/ansible/modules/cloud/digital_ocean/do_sshkeys.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python
```

Comment:
btw, *do_* keeps the same convention from other modules like *cloudstack* = *cs_* ; "openstack" = *os_*